includes/civicrm.admin.php

```diff
@@ -558,6 +561,11 @@ public function settings_change($dao) {
    *
    * @since 5.28
    * @since 5.33 Moved to this class.
+   *
+   * @param string $html The auto-update markup.
+   * @param string $plugin_file The path to the plugin.
+   * @param array $plugin_data An array of plugin data.
+   * @return string $html The modified auto-update markup.
    */
   public function auto_update_prevent($html, $plugin_file, $plugin_data) {
```

Comment:
```suggestion
  public function auto_update_prevent(string $html, string $plugin_file, array $plugin_data): string 
  {
```

Reply:
@altafsamnani Thanks for the suggestion(s) - the plugin doesn't use type declarations anywhere as yet, nor does it enforce them via `declare(strict_types=1)`. To add them throughout the codebase seems like a great commit in its own right but IMO is outside the scope of this PR since type declarations aren't included in the `phpcs` codstyle checks. @kcristiano What do you think?

Reply:
I agree